packages/suite/src/services/suite/invityAPI.ts

```diff
@@ -0,0 +1,277 @@
+import { createHash } from 'crypto';
```

Comment:
not a blocker, but maybe it should be considered to move this class to `invity-api` package

Reply:
I am against it at this moment - `invityAPI.ts` contains code specific for the Suite. The package `invity-api` is used by our additional projects, which have nothing to do with the Suite.

Reply:
that's what i thought at the beginning but i don't see any suite related code in this file either, more like a typed `invity-api` requests

Reply:
You are almost right but there is specific code for headers and api key at this moment. I think we will refactor it together with the code in our other projects and bring it to the Suite later on.